service/src/main/resources/application.yaml

```diff
@@ -12,6 +12,8 @@ management:
 spring:
   application:
     name: fpl-service
+  main:
+    allow-bean-definition-overriding: true
```

Comment:
what is reason for it, for me it smells a bit

Reply:
fixed